aiven/client/cli.py

```diff
@@ -437,6 +437,14 @@ def user__tokens_expire(self) -> None:
         message = self.client.expire_user_tokens()["message"]
         print(message)
 
+    @arg.verbose
+    def user__password_change(self) -> None:
+        """Change the password of the currently logged in user"""
+        current_password = getpass.getpass("Current password:")
+        new_password = self.enter_password("New password:", confirm=True)
```

Comment:
Missing space after colon (in both lines). Also, I would add ` (hidden)` to make it clear that the password is not displayed at all.